Security/src/AuthConsole/Worker.cs

```diff
@@ -0,0 +1,25 @@
+using Steeltoe.Samples.AuthConsole.ApiClients;
+using Steeltoe.Samples.AuthConsole.Models;
+
+namespace Steeltoe.Samples.AuthConsole;
+
+public sealed class Worker(CertificateAuthorizationApiClient certificateAuthorizationApiClient, ILogger<Worker> logger) : BackgroundService
+{
+    protected override async Task ExecuteAsync(CancellationToken cancellationToken)
+    {
+        while (!cancellationToken.IsCancellationRequested)
+        {
+            Console.WriteLine($"Background service starting at: {DateTimeOffset.Now} (press Ctrl+C to close).");
```

Comment:
It would be more helpful to move this to the end. The user easily misses the initial message.

```c#
Console.WriteLine("Sleeping for 10 seconds (press Ctrl+C to close).");
await Task.Delay(10_000, cancellationToken);
```